app.ts

```diff
@@ -872,13 +872,9 @@ async function main() {
             );
         })
         .use(bodyParser.json({limit: ceProps('bodyParserLimit', maxUploadSize)}))
-        .get('/api/siteTemplates', siteTemplateController.getSiteTemplates.bind(siteTemplateController))
-        .get('/source/:source/list', sourceController.listEntries.bind(sourceController))
-        .get('/source/:source/load/:language/:filename', sourceController.loadEntry.bind(sourceController))
-        .get(
-            '/api/asm/:arch/:opcode',
-            assemblyDocumentationController.getOpcodeDocumentation.bind(assemblyDocumentationController),
-        )
+        .use('/', siteTemplateController.createRouter())
```

Comment:
is the `/` necessary? I am never quite sure but it seems like (e.g.) the above bodyParser doesn't need a route?

Reply:
Maybe not, I can do a test without

Reply:
you're right! :star: